zulip/integrations/rss/rss-bot

```diff
@@ -42,6 +42,10 @@ You can customize the location on the feed file and recipient stream, e.g.:
 
 /usr/local/share/zulip/integrations/rss/rss-bot --feed-file=/path/to/my-feeds --stream=my-rss-stream
 
+optionally, you can also specify the name of the stream topic, e.g.:
+
+/usr/local/share/zulip/integrations/rss/rss-bot --feed-file=/path/to/my-feeds --stream=my-rss-stream --topic=my-topic-name
```

Comment:
Maybe specify an example topic containing spaces, e.g. `--topic="topic name"`, since that's more typical naming style.

Reply:
indeed: https://github.com/zulip/python-zulip-api/pull/747/commits/3574ed6493c00e12933c8ccf7256e2d5228c8f22